experiments/AtmosLES/dycoms.jl

```diff
@@ -368,17 +367,20 @@ function main()
 
     xmax = FT(1000)
     ymax = FT(1000)
-    zmax = FT(2500)
+    zmax = FT(1500)
 
     t0 = FT(0)
     timeend = FT(100)
+    Cmax = FT(1.7)     #use this for single-rate explicit LSRK144
+    #Cmax = FT(8.0)     #use this formulti-rate LSRK144
```

Comment:
Since you removed the previously commented code for using Multirate (which perhaps should have stayed), this can be removed too.